### PR TITLE
Added cl_righthand

### LINKED ISSUE #219: cl_righthand

### PR DESCRIPTION
**Pull request by SmileyAG**
Closes #219

# Screenshot

изображение

# Note

On Windows, `IsFlippedViewModel` function been called at **R_GLStudioDrawPoints** itself:

изображение

But, linux compilers of .so binaries somehow turn that function to few checks instead, so that a main reason why I can't rename cvar:

изображение

### CODE REVIEW

**Comment by YaLTeR** [comment]
Maybe just call the cvar `cl_righthand`? Then it will work automatically.

**Comment by YaLTeR** [comment]
¯\_(ツ)_/¯

**Comment by SmileyAG** [comment]
@YaLTeR Review now.

File: BunnymodXT/modules/ClientDLL.cpp [hunk 1]
```diff
@@ -714,6 +714,13 @@ void ClientDLL::RegisterCVarsAndCommands()
 	EngineDevMsg("[client dll] Registering CVars.\n");
 
 	#define REG(cvar) HwDLL::GetInstance().RegisterCVar(CVars::cvar)
+
+	if (!HwDLL::GetInstance().ORIG_Cvar_FindVar(CVars::cl_righthand.GetPointer()->name))
```
Comment (YaLTeR) [inline BunnymodXT/modules/ClientDLL.cpp:718]:
```suggestion
	if (!HwDLL::GetInstance().ORIG_Cvar_FindVar("cl_righthand"))
```
Simpler is better in this case I think

Comment (SmileyAG) [reply BunnymodXT/modules/ClientDLL.cpp]: Done.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**